src/handlers/dispersion.ts

```diff
@@ -81,3 +81,25 @@ export const getTrendingAddresses = async (): Promise<
     return null;
   }
 };
+
+export const getAdditionalAssetData = async (address: EthereumAddress) => {
+  try {
+    const res = await dispersionApi.get(`/dispersion/v1/expanded/1/${address}`);
+    return res?.data?.data;
+  } catch (error) {
+    logger.sentry(`Error fetching additional asset data: ${error}`);
+    captureException(error);
+  }
+};
+
+export const getCoingeckoIds = async (): Promise<
+  Record<EthereumAddress, string> | undefined
+> => {
+  try {
+    const res = await dispersionApi.get('/dispersion/v1/coingecko/ids/1');
+    return res?.data?.data?.ids;
```

Comment:
same here, let's follow the pattern of explicitly return `null` here and in the catch - also to update the return type on line 96 to be `| null` instead of `| undefined`